Vagrantfile

```diff
@@ -89,7 +89,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.define "database" do |database|
     database.vm.hostname = "database"
-    database.hostmanager.aliases = %w(database.service.ashlar.internal)
+    database.hostmanager.aliases = %w(database.service.driver.internal)
```

Comment:
Good call. I'm rebasing your Travis changes too -- need to change references for the settings defaults.